store/src/main/scala/geotrellis/store/util/IORuntimeTransient.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.store.util
+
+import cats.effect._
+
+object IORuntimeTransient extends Serializable {
+  val ThreadsNumber: Int = Runtime.getRuntime.availableProcessors
+  @transient lazy val IORuntime: unsafe.IORuntime = unsafe.IORuntime.global
```

Comment:
We still need a companion object to be sure the runtime init is a `@transient lazy val`, to ensure that it is not serialized and the instance is created per machine.